Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM alpine:3.12
+FROM alpine:3.13
 
-ENV TERRAFORM_VERSION=0.12.25-r0
+ENV TERRAFORM_VERSION=0.13.6
```

Comment:
Why Terraform version is changed to 0.13.6? The latest available is [0.14.5](https://github.com/hashicorp/terraform/releases/tag/v0.14.5)

Reply:
It might involve some other changes we don't want to work on now: https://www.terraform.io/upgrade-guides/0-14.html